mk/vs/Common++.vcxproj.template

```diff
@@ -147,7 +147,7 @@ xcopy "$(PcapPlusPlusHome)\Common++\header\*" "$(PcapPlusPlusHome)\Dist\header"
       <PrecompiledHeader>NotUsing</PrecompiledHeader>
       <WarningLevel>Level3</WarningLevel>
       <Optimization>Disabled</Optimization>
-      <PreprocessorDefinitions>WINx64;MSVC;_CRT_SECURE_NO_WARNINGS;GIT_BRANCH="$(GitBranch)";GIT_COMMIT="$(GitCommit)"</PreprocessorDefinitions>
```

Comment:
I'm don't remember adding these `WINx64` and `MSVC` preprocessor definitions, I think VS adds them by default

Reply:
Maybe it's the default when creating a MS project. But this can be an issue if someone doesn't use Visual Code and include the Pcap++ header in his project the compiler will not define "WINx64" or "WIN32".

So I think it's better to not rely on it, What do you think?

Reply:
Yes, I agree we don't need to rely on it, but I'm not sure we should remove them from all VS projects 🤔

Reply:
Yes but In case they are still some #ifdef WIN32 instead on _WIN32 we will not catch them on the CI.